server/routes/submitvote.js

```diff
@@ -0,0 +1,20 @@
+const express = require("express");
+const router = express.Router();
+const db = require("../database/models");
+const vote_table = db.vote_table;
+
+/**
+ * POST for Vote Submission into vote_table
+ */
+router.post("/", function (req, res, next) {
+  vote_table
+    .bulkCreate(req.body, {
```

Comment:
Maybe something for later. But it would be nice to eventually check the request body to see if it's clean before we enter it into the database :)